db/schema.sql

```diff
@@ -9,28 +9,23 @@ DROP TABLE IF EXISTS plans;
 
 CREATE TABLE users(
     user_id BIGINT NOT NULL AUTO_INCREMENT,
-    first_name VARCHAR(40) NOT NULL,
-    last_name VARCHAR(40) NOT NULL,
-    email VARCHAR(40) NOT NULL UNIQUE,
-    dob DATE NOT NULL,
+    first_name VARCHAR(40),
+    last_name VARCHAR(40),
+    email VARCHAR(40) UNIQUE,
+    dob DATE,
     phone BIGINT NOT NULL UNIQUE,
-    credit_card BIGINT UNIQUE,
+    registration_complete INT(1) NOT NULL,
```

Comment:
We could tell when registration is complete by checking if all the fields are not null. Just concerned we might end up with null fields and `registration_complete = true`, which would be very confusing.